tests/models/idefics/test_modeling_idefics.py

```diff
@@ -337,6 +401,20 @@ def test_generate_with_image_pos_embeddings_interpolation_multiple_images(self):
         )
         self.model_tester.create_and_check_model_gen(*config_and_inputs)
 
+    def test_cross_attention_gates(self):
+        config, inputs_test_1, inputs_test_2 = self.model_tester.prepare_config_and_inputs_gate_tests()
```

Comment:
This would benefit from clearer var naming and/or comments to explain what the expected behaviour is e.g. `inputs_w_image_attn`